cbrkit/sim/collections.py

```diff
@@ -190,3 +192,227 @@ def wrapped_func(query: Sequence[ValueType], case: Sequence[ValueType]) -> float
         return best_score
 
     return wrapped_func
+
+
+@dataclass
+class ListMappingSim:
+    value: float
+    local_similarities: List[float] = None
+
+
+def list_mapping(
+    element_similarity: Callable[[Any, Any], float] = None,
+    exact: bool = False,
+    return_local_similarities: bool = False,
```

Comment:
Is this parameter still needed? Now that a dataclass is used (good!), the value could always be added. Or is there some performance impact?